frontend/src/styles/components/angular-material/mat-select.sass

```diff
@@ -0,0 +1,10 @@
+﻿::ng-deep mat-select
+    align-items: center !important
+    display: flex !important
+
+.mat-select
+    font-size: 14px
+    padding-left: 5px
+    border: 2px solid $pattens-blue
+    border-radius: 8px
+    height: 36px
```

Comment:
I think it's not a good idea to have padding, border etc. set globally, please use ng-deep with custom classes in your component

Reply:
fix